src/cluster/placement/algo/mirrored_test.go

```diff
@@ -701,8 +504,8 @@ func TestMirrorReplaceAndRevertBeforeCutover(t *testing.T) {
 	assert.NoError(t, err)
 	assert.NoError(t, placement.Validate(p1))
 	assert.Equal(t, uint32(2), p1.MaxShardSetID())
-	assert.True(t, allLeaving(p1, []placement.Instance{i4}, nowNanos))
-	assert.True(t, allInitializing(p1, []string{"i5"}, nowNanos))
+	assert.True(t, globalChecker.allLeaving(p1, []placement.Instance{i4}, nowNanos))
+	assert.True(t, localChecker.allInitializing(p1, []string{"i5"}, nowNanos))
```

Comment:
Just sanity checking--this is an intentional behavior change here because previously `Replace` had the (unneeded) sideeffect of `MarkAllShardsAvailable`, but now it does not. Sound right?

Reply:
Exactly.